utests/tests/ptf/test_mask.py

```diff
@@ -12,7 +14,7 @@ def test_mask__mask_simple_packet(self):
         modified_packet = Ether() / IP() / TCP(sport=97)
```

Comment:
If tests are using these default headers, you can use them as module fixtures

Reply:
Please look into: https://github.com/p4lang/ptf/pull/165#discussion_r777708362